frontend/src/app/task-area/task-area.component.scss

```diff
@@ -0,0 +1,238 @@
+.wrapper {
```

Comment:
This file is SASS but does not use it key feature as it should be (nesting elements)